node/src/components/consensus/era_supervisor.rs

```diff
@@ -643,19 +643,13 @@ where
                 equivocators,
                 proposer,
             }) => {
+                self.era_mut(era_id).add_accusations(&equivocators);
+                self.era_mut(era_id).add_accusations(value.accusations());
```

Comment:
`value.accusations` includes equivocators from the past eras. right? We will filter them out in `add_accusations` vis `!slashed.contains(vid)`.

Reply:
Exactly.

Reply:
So `value.accusations()` will have other equivocators, not only from the past era(s)?

Reply:
It _could_. We don't enforce that it does or doesn't have them. But if a block _does_ accuse someone, it depends on evidence.

Reply:
And this PR ensures that if _any_ block in an era accused you, you get slashed at the end.

Reply:
I understand. I am now confused which accusations go to the `ConsensusValue` and which are part of the `FinalizedBlock`.

Reply:
The way the era supervisor is implemented right now, you add everything your Highway instance can see to `accusations`. This is a bit redundant, since all of those validators are guaranteed to also show up in each node's `FinalizedBlock` once your value gets finalized.
However, you _also_ add to your accusations any not-yet-slashed equivocators from _previous_ eras. Those won't show up in anyone's `FinalizedBlock`, but create a new dependency of your value (evidence from previous eras).